src/components/Pagination/__tests__/Pagination-test.js

```diff
@@ -13,14 +13,17 @@ describe('Pagination', () => {
   const defaultProps = {
     cssClasses: {
       root: 'root',
+      noRefinementRoot: 'noRefinementRoot',
+      list: 'list',
       item: 'item',
-      page: 'page',
-      previous: 'previous',
-      next: 'next',
-      first: 'first',
-      last: 'last',
-      active: 'active',
-      disabled: 'disabled',
+      firstPageItem: 'firstPageItem',
+      lastPageItem: 'lastPageItem',
+      previousPageItem: 'previousPageItem',
+      nextPageItem: 'nextPageItem',
+      pageItem: 'pageItem',
```

Comment:
Why not `item` instead of `pageItem`? It would be more consistent.

Reply:
I based this name on the CSS class `ais-Pagination-item--page` → `pageItem`.

Another example of a modifier would be `ais-Pagination-item--firstPage` → `firstPageItem`.


Should we just go for `item` in this specific case, then?